src/main/java/com/example/bankservicedemo/controller/UserController.java

```diff
@@ -0,0 +1,53 @@
+package com.example.bankservicedemo.controller;
+
+import com.example.bankservicedemo.mapper.UserMapper;
+import com.example.bankservicedemo.model.dto.user.UserRequestDto;
+import com.example.bankservicedemo.model.dto.user.UserResponseDto;
+import com.example.bankservicedemo.service.UserService;
+import java.util.List;
+import java.util.stream.Collectors;
+import lombok.AllArgsConstructor;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/users")
+@AllArgsConstructor
+public class UserController {
+    private final UserService userService;
+    private final UserMapper userMapper;
+
+    @PostMapping
+    public void addNewUser(UserRequestDto userRequestDto) {
+        userService.create(userMapper.toEntity(userRequestDto));
+    }
+
+    @GetMapping("/update")
+    public void updateUser(UserRequestDto userRequestDto) {
```

Comment:
why GET?
please see the task description and URL mappings that should be created
https://mate-academy.github.io/jv-program-fulltime/05_spring_boot/bank_service.html